keyboards/handwired/qc60/proto/proto.h

```diff
@@ -0,0 +1,201 @@
+    {XXX, R36, R35, R34, R33, R32, R31, XXX}, \
+    {R47, R46, R45, R44, R43, R42, XXX, XXX}, \
+}
+
+#define LAYOUT_ANSI_ALT( \
+    L00, L01, L02, L03, L04, L05, L06, R00, R01, R02, R03, R04, R05,      R07, \
+    L10, L11, L12, L13, L14, L15,      R10, R11, R12, R13, R14, R15, R16, R17, \
+    L20, L21, L22, L23, L24, L25,           R21, R22, R23, R24, R25, R26, R27, \
+         L31, L32, L33, L34, L35, L36,      R31, R32, R33, R34, R35, R36,      \
+    L40, L41, L42, L43, L44,                     R43, R44,      R45, R46, R47  \
+) \
+{ \
+    {L00, L01, L02, L03, L04, L05, L06, XXX}, \
+    {L10, L11, L12, L13, L14, L15, XXX, XXX}, \
+    {L20, L21, L22, L23, L24, L25, XXX, XXX}, \
+    {XXX, L31, L32, L33, L34, L35, L36, XXX}, \
+    {L40, L41, L42, L43, L44, XXX, XXX, XXX}, \
+    {R07, XXX, R05, R04, R03, R02, R01, R00}, \
+    {R17, R16, R15, R14, R13, R12, R11, R10}, \
+    {R27, R26, R25, R24, R23, R22, R21, XXX}, \
+    {XXX, R36, R35, R34, R33, R32, R31, XXX}, \
+    {R47, R46, R45, R44, R43, XXX, XXX, XXX}, \
+}
+
+#define LAYOUT_ISO_DEFAULT( \
+    L00, L01, L02, L03, L04, L05, L06, R00, R01, R02, R03, R04, R05,           R07, \
+    L10, L11, L12, L13, L14, L15,      R10, R11, R12, R13, R14, R15, R16,           \
+    L20, L21, L22, L23, L24, L25,           R21, R22, R23, R24, R25, R26, R17, R27, \
+         L31, L32, L33, L34, L35, L36,      R31, R32, R33, R34, R35, R36,           \
+    L40, L41, L42, L43, L44,                     R43, R44, R42, R45, R46,      R47  \
+) \
+{ \
+    {L00, L01, L02, L03, L04, L05, L06, XXX}, \
+    {L10, L11, L12, L13, L14, L15, XXX, XXX}, \
+    {L20, L21, L22, L23, L24, L25, XXX, XXX}, \
+    {XXX, L31, L32, L33, L34, L35, L36, XXX}, \
+    {L40, L41, L42, L43, L44, XXX, XXX, XXX}, \
+    {R07, XXX, R05, R04, R03, R02, R01, R00}, \
+    {R17, R16, R15, R14, R13, R12, R11, R10}, \
+    {R27, R26, R25, R24, R23, R22, R21, XXX}, \
+    {XXX, R36, R35, R34, R33, R32, R31, XXX}, \
+    {R47, R46, R45, R44, R43, R42, XXX, XXX}, \
+}
+
+#define LAYOUT_ISO_ALT( \
+    L00, L01, L02, L03, L04, L05, L06, R00, R01, R02, R03, R04, R05,      R07, \
+    L10, L11, L12, L13, L14, L15,      R10, R11, R12, R13, R14, R15, R16, R17, \
+    L20, L21, L22, L23, L24, L25,           R21, R22, R23, R24, R25, R26, R27, \
+         L31, L32, L33, L34, L35, L36,      R31, R32, R33, R34, R35, R36,      \
+    L40, L41, L42, L43, L44,                     R43, R44,      R45, R46, R47  \
+) \
+{ \
+    {L00, L01, L02, L03, L04, L05, L06, XXX}, \
+    {L10, L11, L12, L13, L14, L15, XXX, XXX}, \
+    {L20, L21, L22, L23, L24, L25, XXX, XXX}, \
+    {XXX, L31, L32, L33, L34, L35, L36, XXX}, \
+    {L40, L41, L42, L43, L44, XXX, XXX, XXX}, \
+    {R07, XXX, R05, R04, R03, R02, R01, R00}, \
+    {R17, R16, R15, R14, R13, R12, R11, R10}, \
+    {R27, R26, R25, R24, R23, R22, R21, XXX}, \
+    {XXX, R36, R35, R34, R33, R32, R31, XXX}, \
+    {R47, R46, R45, R44, R43, XXX, XXX, XXX}, \
+}
+
+#define LAYOUT_HHKB_DEFAULT( \
```

Comment:
This should be `LAYOUT_hhkb_default`

Does this match the actual HHKB layout?

Reply:
Because it's a split keyboard, the bottom row doesn't 100% follow the HHKB layout. The layout macro, along with splitting backspace and right shift, does not include the leftmost and rightmost keys on the bottom row which is the same behavior with HHKB layouts.